core/utils/http.go

```diff
@@ -13,6 +13,24 @@ import (
 	"github.com/smartcontractkit/chainlink/core/logger"
 )
 
+var (
+	Client             *http.Client
```

Comment:
do we need to export these?

Reply:
I was hoping to re-use them in this PR: https://github.com/smartcontractkit/chainlink/pull/3859